packages/react-core/src/next/components/Wizard/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './Wizard';
+export * from './WizardContext';
+export * from './WizardBody';
+export * from './WizardFooter';
+export * from './WizardToggle';
+export * from './WizardStep';
+export { useWizardFooter } from './hooks/useWizardFooter';
+export * from './types';
```

Comment:
If we are going to export all the types her for consumption, they should be documented on the website.